clang/include/clang/Sema/SemaInternal.h

```diff
@@ -58,7 +58,7 @@ inline InheritableAttr *getDLLAttr(Decl *D) {
 }
 
 /// Retrieve the depth and index of a template parameter.
-inline std::pair<unsigned, unsigned> getDepthAndIndex(NamedDecl *ND) {
+inline std::pair<unsigned, unsigned> getDepthAndIndex(const NamedDecl *ND) {
```

Comment:
This could be an nfc change

Reply:
I'll split this out into a separate patch

Reply:
Done, see https://github.com/llvm/llvm-project/pull/113857